src/main/scala/org/broadinstitute/hail/driver/Command.scala

```diff
@@ -3,17 +3,21 @@ package org.broadinstitute.hail.driver
 import org.apache.spark.SparkContext
 import org.apache.spark.sql.SQLContext
 import org.broadinstitute.hail.utils._
-import org.broadinstitute.hail.variant.VariantDataset
+
 import org.kohsuke.args4j.{Argument, CmdLineException, CmdLineParser, Option => Args4jOption}
 
+import org.broadinstitute.hail.variant.{HardCallSet, VariantDataset}
+
 import scala.collection.JavaConverters._
+
 import scala.collection.mutable
 
 case class State(sc: SparkContext,
   sqlContext: SQLContext,
   // FIXME make option
   vds: VariantDataset = null,
-  env: Map[String, VariantDataset] = Map.empty) {
+  env: Map[String, VariantDataset] = Map.empty,
+  hcs: HardCallSet = null) {
```

Comment:
My instinct is for this to be `Option[HardCallSet]` rather than to introduce more `null`s to our system.